examples/fluent-bit/amazon-opensearch-serverless/README.md

```diff
@@ -0,0 +1,21 @@
+### FireLens Example: Logging to Amazon OpenSearch Serverless Service with Fluent Bit
+
+Amazon OpenSearch Serverless is new offering that eliminates your need to manage OpenSearch instances. All existing Fluent Bit OpenSearch output plugin options work with OpenSearch Serverless. The only difference with serverless from a Fluent Bit POV is that you must specify the service name as aoss (Amazon OpenSearch Serverless) when you enable AWS_Auth:
```

Comment:
Can you put `aoss` in backticks, same with `AWS_Auth`?

Reply:
Changed